tracker/views.py

```diff
@@ -21,8 +21,12 @@
 from django.contrib.auth.forms import AuthenticationForm
 from django.contrib import messages
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import LoginView    
 
+class Login(LoginView):
+    template_name = 'Login.html'
```

Comment:
What's this about?

Reply:
Oh, I need to delete this, I thought that I needed this at one point when I trying to fix the error with the login page. I forgot to take it out